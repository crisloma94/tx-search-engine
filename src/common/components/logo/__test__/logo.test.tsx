import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import Theme from '../../../../styles/theme';
import { store } from '../../../../core/store/store';
import Logo from '../logo.component';

describe('Logo component test suite', () => {
  const getComponent = () => (
    <Provider store={store}>
      <Theme>
        <Logo />
      </Theme>
    </Provider>
  );
  const setup = () => render(getComponent());

  it('Should match snapshot', () => {
    const { container } = setup();
    expect(container).toMatchSnapshot();
  });
});
