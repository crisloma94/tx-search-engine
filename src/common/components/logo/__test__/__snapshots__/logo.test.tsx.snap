// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component test suite Should match snapshot 1`] = `
<div>
  <img
    alt="Logo"
    src="test-file-stub"
  />
  <span>
    HappyBooks
  </span>
</div>
`;
